socrate/system.py

```diff
@@ -18,3 +19,14 @@ def resolve_address(address):
     hostname, *rest = address.rsplit(":", 1)
     ip_address = resolve_hostname(hostname)
     return ip_address + "".join(":" + port for port in rest)
+
+
+def get_host_address_from_environment(name, default):
+    """ This function looks up an envionment variable ``{{ name }}_ADDRESS``.
+    If it's defined, it is returned unmodified. If it's undefined, an environment
+    variable ``HOST_{{ name }}`` is looked up and resolved to an ip address.
+    If this is also not defined, the default is resolved to an ip address.
+    """
+    if "{}_ADDRESS".format(name) in environ:
+        return environ.get("{}_ADDRESS".format(name))
+    return resolve_address(environ.get("HOST_{}".format(name),default))
```

Comment:
incredible nitpick: space after comma :sweat_smile: . I almost feel sorry for pointing out.

Reply:
No worries, fixed.